active/0008-community-plugins.md

```diff
@@ -24,14 +29,26 @@ where we list approved community plugins.
 which may otherwise cause more lines of conflicts when porting changes to enterprise.)
 
 ```erlang
-[ {foo_plugin, {git, "https://github.com/###"}}
-, {bar_plugin, {git, "https://github.com/###"}} ].
+deps() ->
```

Comment:
if this file is intended for Erlang's `file:consult` API. we can't write functions.
maybe keep it simple as 
```
{erlang_plugins, [{foo_plugin, {git, "https://github.com", {tag, "1.0.0"}}}}]}.
{elixir_plugins, [{bar_plugin, {git, "https://github.com", {tag, "1.0.0"}}}}]}.
```

Reply:
updated